index.src.html

```diff
@@ -70,6 +70,9 @@
       text: Navigator
 spec: HTML; urlPrefix: https://html.spec.whatwg.org/multipage/
   type: dfn
+    text: same-origin; url: concept-origin-effective-domain
+    text: origin; url: concept-origin
+    text: ASCII serialization; url: ascii-serialisation-of-an-origin
```

Comment:
How many times do we use this? I think I'd prefer to just have `lt="ASCII serialization of an origin"` in the markup than to add another inline anchor. I'd prefer that because if we add the anchor inline, then we're responsible for keeping it up to date as the external spec shifts around us. I'd prefer to rely on the magical autolinking system rather than taking responsibility ourselves....

This, of course, isn't terribly likely to shift out from under us, but it's the principle of the thing!